src/linear_algebra/std_basis.lean

```diff
@@ -250,6 +251,11 @@ by { simp only [basis_fun, basis.coe_of_equiv_fun, linear_equiv.refl_symm,
   (pi.basis_fun R η).repr x i = x i :=
 by simp [basis_fun]
 
+/-- The standard basis on `matrix n m R`. -/
+noncomputable def _root_.matrix.std_basis (n : Type*) (m : Type*) [fintype m] [fintype n] :
+  basis (Σ (j : n), (λ (i : n), m) j) R (matrix n m R) :=
```

Comment:
It would be nice if we could use the more natural `n \times m` index here, but that's fine for a follow-up. I would guess you can just send this basis along an equiv?

Reply:
Yes, that's should be easy. It's now in my TODO list :)